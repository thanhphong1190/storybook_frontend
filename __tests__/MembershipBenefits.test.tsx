import renderer from 'react-test-renderer'
import {
  MembershipBenefits,
  MembershipBenefitsProps
} from '../src/components/membership-benefits/MembershipBenefits'

const props: MembershipBenefitsProps = {
  title: 'Membership benefits',
  text: `Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et`,
  benefits: [
    {
      text: `All Strength & Cardio Equipments`,
      iconImgURL: '/example/benefits/strength-cardio-equipment.original.png'
    },
    {
      text: `Unlimited Virtual Classes`,
      iconImgURL: '/example/benefits/unlimited-virtual-classes.png'
    },
    {
      text: `24 hr access, 7 Days a Week`,
      iconImgURL: '/example/benefits/24-7.original.png'
    },
    {
      text: `No Lock In Contracts`,
      iconImgURL: '/example/benefits/no-lock-in-contracts.original.png'
    },
    {
      text: `FREE Fitness Programs`,
      iconImgURL: '/example/benefits/free-fitness-program.original.png'
    },
    {
      text: `Access all 180+ gyms in AU & NZ`,
      iconImgURL: '/example/benefits/access.original.png'
    }
  ]
}

it(`Membership benefits renders correctly`, () => {
  const tree = renderer.create(<MembershipBenefits {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
