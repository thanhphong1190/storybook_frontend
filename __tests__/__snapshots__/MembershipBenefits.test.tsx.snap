// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Membership benefits renders correctly 1`] = `
<section
  className="MembershipBenefitsstyles__Section-sc-9ajgpb-0 zcvVr"
>
  <div
    className="MembershipBenefitsstyles__Container-sc-9ajgpb-1 exAHc"
  >
    <div
      className="MembershipBenefitsstyles__TitleWrapper-sc-9ajgpb-3 iWYBSd"
    >
      <div
        className="globalstyles__Text-sc-1nc69wi-9 dBcsmZ"
        color="orange"
        fontWeight="bold"
        size="4xl"
      >
        Membership benefits
      </div>
      <hr
        className="globalstyles__Underline-sc-1nc69wi-8 gfaeFo"
        color="blue"
      />
    </div>
    <div
      className="MembershipBenefitsstyles__TextWrapper-sc-9ajgpb-4 fPZvhX"
    >
      <div
        className="globalstyles__Text-sc-1nc69wi-9 dvGdRc"
        color="white"
      >
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et
      </div>
    </div>
    <div
      className="MembershipBenefitsstyles__BenefitsContainer-sc-9ajgpb-5 lomQdr"
    >
      <div
        className="MembershipBenefitsstyles__BenefitsWrapper-sc-9ajgpb-6 dROzFS"
      >
        <div
          className="MembershipBenefitsstyles__Benefit-sc-9ajgpb-7 cysZvh"
        >
          <div
            className="MembershipBenefitsstyles__ImgWrapper-sc-9ajgpb-8 iaKkPN"
          >
            <img
              className="MembershipBenefitsstyles__Img-sc-9ajgpb-2 kPKlve"
              src="/example/benefits/strength-cardio-equipment.original.png"
            />
          </div>
          <span
            className="MembershipBenefitsstyles__BenefitText-sc-9ajgpb-9 cDTXJl"
          >
            All Strength & Cardio Equipments
          </span>
        </div>
        <div
          className="MembershipBenefitsstyles__Benefit-sc-9ajgpb-7 cysZvh"
        >
          <div
            className="MembershipBenefitsstyles__ImgWrapper-sc-9ajgpb-8 iaKkPN"
          >
            <img
              className="MembershipBenefitsstyles__Img-sc-9ajgpb-2 kPKlve"
              src="/example/benefits/unlimited-virtual-classes.png"
            />
          </div>
          <span
            className="MembershipBenefitsstyles__BenefitText-sc-9ajgpb-9 cDTXJl"
          >
            Unlimited Virtual Classes
          </span>
        </div>
        <div
          className="MembershipBenefitsstyles__Benefit-sc-9ajgpb-7 cysZvh"
        >
          <div
            className="MembershipBenefitsstyles__ImgWrapper-sc-9ajgpb-8 iaKkPN"
          >
            <img
              className="MembershipBenefitsstyles__Img-sc-9ajgpb-2 kPKlve"
              src="/example/benefits/24-7.original.png"
            />
          </div>
          <span
            className="MembershipBenefitsstyles__BenefitText-sc-9ajgpb-9 cDTXJl"
          >
            24 hr access, 7 Days a Week
          </span>
        </div>
        <div
          className="MembershipBenefitsstyles__Benefit-sc-9ajgpb-7 cysZvh"
        >
          <div
            className="MembershipBenefitsstyles__ImgWrapper-sc-9ajgpb-8 iaKkPN"
          >
            <img
              className="MembershipBenefitsstyles__Img-sc-9ajgpb-2 kPKlve"
              src="/example/benefits/no-lock-in-contracts.original.png"
            />
          </div>
          <span
            className="MembershipBenefitsstyles__BenefitText-sc-9ajgpb-9 cDTXJl"
          >
            No Lock In Contracts
          </span>
        </div>
        <div
          className="MembershipBenefitsstyles__Benefit-sc-9ajgpb-7 cysZvh"
        >
          <div
            className="MembershipBenefitsstyles__ImgWrapper-sc-9ajgpb-8 iaKkPN"
          >
            <img
              className="MembershipBenefitsstyles__Img-sc-9ajgpb-2 kPKlve"
              src="/example/benefits/free-fitness-program.original.png"
            />
          </div>
          <span
            className="MembershipBenefitsstyles__BenefitText-sc-9ajgpb-9 cDTXJl"
          >
            FREE Fitness Programs
          </span>
        </div>
        <div
          className="MembershipBenefitsstyles__Benefit-sc-9ajgpb-7 cysZvh"
        >
          <div
            className="MembershipBenefitsstyles__ImgWrapper-sc-9ajgpb-8 iaKkPN"
          >
            <img
              className="MembershipBenefitsstyles__Img-sc-9ajgpb-2 kPKlve"
              src="/example/benefits/access.original.png"
            />
          </div>
          <span
            className="MembershipBenefitsstyles__BenefitText-sc-9ajgpb-9 cDTXJl"
          >
            Access all 180+ gyms in AU & NZ
          </span>
        </div>
      </div>
    </div>
  </div>
</section>
`;
